code/controllers/subsystem/profiler.dm

```diff
@@ -51,7 +51,7 @@ SUBSYSTEM_DEF(profiler)
 	// FETCH PROC PROFILE //
 
 	// Fetch info
-	var/current_profile_data = world.Profile(PROFILE_REFRESH, format = "json")
+	var/current_profile_data = world.Profile(PROFILE_REFRESH, "json")
 	nfetch_cost = MC_AVERAGE(nfetch_cost, TICK_DELTA_TO_MS(TICK_USAGE_REAL - timer))
 	CHECK_TICK // this shouldnt sleep given its being called from fire() but ehhhhhhhhhhhhhhhh
```

Comment:
This is fubar until we figure out what the heck is going on with it.

If I use named args it whines at the named args. 
```
[2022-11-07T22:50:01] Runtime in profiler.dm,54: illegal use of list2args() or named parameters
  proc name: DumpFile (/datum/controller/subsystem/profiler/proc/DumpFile)
  src: Profiler (/datum/controller/subsystem/profiler)
  call stack:
  Profiler (/datum/controller/subsystem/profiler): DumpFile()
  Profiler (/datum/controller/subsystem/profiler): Shutdown()
  Master (/datum/controller/master): Shutdown()
  world: Reboot(null, 0)
  Ticker (/datum/controller/subsystem/ticker): reboot helper("Unit Test Reboot", "tests ended", 0)
  /datum/test_runner (/datum/test_runner): Run()
  Ticker (/datum/controller/subsystem/ticker): setup()
  Ticker (/datum/controller/subsystem/ticker): fire(0)
  Ticker (/datum/controller/subsystem/ticker): ignite(0)
```

If I dont use named args it whines at a lack of data
```
[2022-11-07T22:55:35] Runtime in stacktrace.dm,14: Warning, profiling stopped manually before dump.
  proc name: stack trace (/datum/proc/stack_trace)
  src: Profiler (/datum/controller/subsystem/profiler)
  call stack:
  Profiler (/datum/controller/subsystem/profiler): stack trace("Warning, profiling stopped man...")
  Profiler (/datum/controller/subsystem/profiler): DumpFile()
  Profiler (/datum/controller/subsystem/profiler): fire(0)
  Profiler (/datum/controller/subsystem/profiler): ignite(0)
```

I have no idea